bindings/python/py_src/safetensors/flax.py

```diff
@@ -109,6 +109,10 @@ def load_file(filename: Union[str, os.PathLike]) -> Dict[str, jnp.DeviceArray]:
             The device where the tensors need to be located after load.
             available options are all regular flax device locations
 
+            Note: In future versions, the default will be removed. The device
+            is required to be set to the real final device to get the best
+            load performance. Default will be removed in version >= 0.5.
```

Comment:
```suggestion
            <Tip warning={true}>
            
            Explicitly setting the device is strongly recommended to get the best
            load performance. In version >= 0.5, not setting `device` will result in an error.
            
             </Tip> 
```
doc-builder syntax + better message.